src/Symfony/Component/Cache/Traits/MemcachedTrait.php

```diff
@@ -31,6 +31,12 @@ trait MemcachedTrait
         \Memcached::OPT_SERIALIZER => \Memcached::SERIALIZER_PHP,
     ];
 
+    /**
+     * We are replacing characters that are illegal in Memcached keys with reserved characters from
+     * {@see \Symfony\Contracts\Cache\ItemInterface::RESERVED_CHARACTERS} that are legal in Memcached.
+     */
+    private static $encoding = ['memcached' => " \n\r\t\v\f", 'symfony' => '@():{}'];
```

Comment:
let's use two private consts :)

Reply:
Traits cannot have constants

Reply:
let's use two private static vars, we'll turn them into consts in v5, where the trait has been removed

Reply:
updated